src/main/scala/io/citrine/lolo/bags/MultiTaskBagger.scala

```diff
@@ -110,23 +111,68 @@ class MultiTaskBaggedTrainingResult(
 
   lazy val model = new MultiTaskBaggedModel(models, Nib, useJackknife, biasModels)
 
+  lazy val predictedVsActual: Seq[(Vector[Any], Seq[Option[Any]], Seq[Option[Any]])] =
```

Comment:
Slinging tuples is hard to read. Going forward, would be nice to use something more specialized so that we can write e.g. `_.actual` instead of `_._3`. For now, let's shore this up a little by adding a comment here that documents what each item in the tuple means. (It's somewhat self-documenting at the end of this var assignment block, but some prose before line 114 would be choice.)

Reply:
I forgot to add these comments before, but I just added them. Because the merge is into a feature branch, I don't need re-review